### ISSUE TITLE
Cheat Discontinued ?

### ISSUE THREAD

**Issue by spookykokojunge** [IntelSDM/HuntDMA#16]
Hey, did you discontinue the Hunt cheat? Because there are still some major bugs left, and you're not updating the cheat anymore. So, did you abandon it, or will there be updates in the future?

**Comment by IntelSDM** [comment]
I might touch upon it in the future but I am submerged up to my neck in work. I just checked UC and someone posted a sig for systemglobalenvironment that should be good for quite a long while.

**Comment by spookykokojunge** [comment]
Yes, I got my offset from there. Additionally, I added all the bosses to the ESP and removed the zombies from the monster ESP, making it much easier to spot the bosses. However, I'm currently stuck on removing your custom GUI so I can replace it with ImGui and have a transparent overlay for my Hunt. I'm running it on a single PC with DMA without any issues, similar to how I manage it for Apex ,Tarkovand others.

I understand you're swamped with work right now and can't continue updating the cheat. Any tips or pointers you can provide to help with the GUI issue would be greatly appreciated. Your work has been incredibly valuable, and any small guidance would help me keep improving it in your absence.
